pkg/acsengine/const.go

```diff
@@ -15,12 +15,16 @@ const (
 	DefaultFirstConsecutiveKubernetesStaticIP = "10.240.255.5"
 	// DefaultAgentSubnetTemplate specifies a default agent subnet
 	DefaultAgentSubnetTemplate = "10.%d.0.0/16"
```

Comment:
Now with CNI, we can max out our nodes.  There are 12 agent pools, and each can have 100 nodes.  When CNI is chosen, we should add these under a separate range from the master IPs, and make that range support 78,000, perhaps a /15 (12 pools * 100 machines * 65 addresses per host (machine + 64) == 76,800).